showcase.json

```diff
@@ -1,6 +1,21 @@
-[
+[{
+    "id": "motokolearn",
+    "name": "MOTOKOLEARN",
+    "oneLiner": "Motoko package meant to facilitate on-chain training and inference of machine learning models where having a large GPU is not a requirement",
+    "website": "https://github.com/ildefons/motokolearn",
+    "tags": [
+      "Tools / Infrastructure"
+    ],
+    "twitter": "https://twitter.com/ildefons",
+    "description": "Motoko package meant to facilitate on-chain training and inference of machine learning models where having a large GPU is not a requirement",
+    "usesInternetIdentity": false,
+    "stats": "",
+    "display": "",
+    "logo": "",
```

Comment:
Can you please provide a logo, screenshot, and link to a video demo (you can add the `video` key value) on how this works?

Reply:
This is a motoko package to train machine learning models. I don't have any logo and I don't have any video because the best way to learn is to read the tuturials  here: https://github.com/ildefons/motokolearn/blob/master/README.md